crates/re_types/src/components/disconnected_space_ext.rs

```diff
@@ -0,0 +1,7 @@
+use super::DisconnectedSpace;
+
+impl From<bool> for DisconnectedSpace {
+    fn from(b: bool) -> Self {
+        Self(b)
+    }
+}
```

Comment:
The UX on the API would be pretty awful otherwise because this is both an archetype and a component within that archetype (`rerun::DisconnectedSpace::new(rerun::datatypes::DisconnectedSpace(true))` vs `rerun::DisconnectedSpace::new(true)`.

I can add a default value yep.